scripts/generate_maps.py

```diff
@@ -1,97 +1,118 @@
 
-# Define the paths
-input_file_path = "./antarctic_index.gpkg"
-map_path = "/Users/nathanbekele/Downloads/Quantarctica3/Miscellaneous/SimpleBasemap/ADD_DerivedLowresBasemap.shp"
-output_folder = "./data3"
-statistics_folder = "./data_statistics"
+def determine_availability(row):
+    if row['availability'] == 's':
+        return 'yes'
+    elif row['availability'] == 'u':
+        return 'no'
+    else:
+        return 'outdated'
 
-# Ensure the output folders exist
-os.makedirs(output_folder, exist_ok=True)
-os.makedirs(statistics_folder, exist_ok=True)
+def plot_data_on_basemap(basemap, gdf, institution, filename, output_folder):
+    fig, ax = plt.subplots(figsize=(10, 10))
 
-# Read the GeoPackage file into a GeoDataFrame
-fp = gpd.read_file(input_file_path)
+    # Set background color to white
+    fig.patch.set_facecolor('white')
+    ax.set_facecolor('white')
 
-# Read the map shapefile
-mp = gpd.read_file(map_path)
+    # Plot basemap with custom colors
+    basemap.plot(ax=ax, color=basemap['color'], edgecolor='black')
 
-# Check if the 'Category' column exists
-if 'Category' in mp.columns:
-    # Define the colors for the categories
-    category_colors = {
-        'Ocean': '#a3bdd1',
-        'Ice shelf': '#cfe1eb',
-        'Land': '#f0f0f0',
-        'Sub-antarctic_G': 'lightgreen',
-        'Sub-antarctic_L': 'lightblue',
-        'Ice tongue': 'lightgrey',
-        'Rumple': 'yellow',
-    }
+    # Define colors and labels based on availability
+    availability_colors = {'u': '#fb9a99', 's': '#1f78bc', 'o': 'grey'}
+    availability_labels = {'u': 'Unavailable', 's': 'Available', 'o': 'Outdated'}
 
-    # Create a color column based on the 'Category' column
-    mp['color'] = mp['Category'].map(category_colors).fillna('grey')  # Default to grey for undefined categories
+    # Plot the data for the institution
+    if not gdf.empty:
+        for availability in ['s', 'u', 'o']:  # Ensure the order of plotting
+            subset = gdf[gdf['availability'] == availability]
+            if not subset.empty:
+                color = availability_colors.get(availability, 'darkgrey')
+                label = availability_labels.get(availability, 'Other')
+                subset.plot(ax=ax, color=color, markersize=0.55, linewidth=0.25, label=label)
+
+    # Set limits to zoom in on Antarctica
+    ax.set_xlim(-3e6, 3e6)
+    ax.set_ylim(-3e6, 3e6)
 
-    # Function to plot data on the colored basemap
-    def plot_data_on_basemap(basemap, gdf, institution, filename):
-        fig, ax = plt.subplots(figsize=(10, 10))
+    # Set equal aspect ratio for circular plot and remove axes
+    ax.set_aspect('equal')
+    ax.axis('off')
 
-        # Set background color to white
-        fig.patch.set_facecolor('white')
-        ax.set_facecolor('white')
+    # Add custom scale bar with "1000 km" label only
+    scalebar = ScaleBar(1, location='lower right', units='km', dimension='si-length', label='1000 km', length_fraction=0.2)
```

Comment:
I'm not familiar with this artist, so don't know off the top of my head what your mistake is, but the "1Gm" labels on the plots are wrong. (1000 km is sufficient; other/additional length units are non standard)